data/human/free worlds side plots.txt

```diff
@@ -808,3 +808,74 @@ mission "FW Stack Core 1C"
+	minor
+	source
+		attributes "dirt belt" rim south
+		not government Quarg
+		not near Tarazed 1
+	destination Humanika
+	to offer
+		has "event: war begins"
+		not "chosen sides"
+		random < 15
+	to fail
+		has "event: joined the free worlds"
+	passengers 3
+	cargo "household goods" 2
+	on offer
+		conversation
+			`A young family of <bunks> is walking around the spaceport talking to merchant captains. From the looks of it, they're not having much luck.`
+			choice
+				`	(Approach them.)`
+				`	(Ignore them.)`
+					defer
+			`	The parents introduce themselves as Penny Little and Rosa Perkins, and their young child, Eduardo. "Any chance you're willing to take us and <tons> of our belongings to the Quarg planet <destination>? We're concerned that the war between the Republic and the Free Worlds is going to get dangerous, and we want to raise Eduardo somewhere safe. We can pay <payment>."`
+			label "main choice"
+			choice
+				`	"Why were all those other captains refusing to take you?"`
+					to display
+						not "FW Refugees to Humanika: other captains"
+					goto "other captains"
+				`	"What do you mean, 'Quarg planet'?"`
+					to display
+						not "FW Refugees to Humanika: quarg"
+						not "First Contact: Quarg: offered"
+					goto quarg
+				`	"The Quarg let humans live on one of their planets?"`
+					to display
+						not "FW Refugees to Humanika: quarg"
+						has "First Contact: Quarg: offered"
+						not "visited planet: Humanika"
+					goto quarg
+				`	"Sure, I can take you."`
+					accept
+				`	"Sorry, I'm not headed that way."`
+					decline
+			label "other captains"
+			action
+				set "FW Refugees to Humanika: other captains"
+			`	"I think many of them are a bit distrustful of the Quarg, and people are just generally more on edge these days with the tensions between the Free Worlds and the Republic, so they're less interested in flying somewhere unfamiliar. While I do understand those feelings, it seems to me that humans have done much worse to each other than the Quarg have ever done to us."`
+				goto "main choice"
+			label "quarg"
+			action
+				set "FW Refugees to Humanika: quarg"
+			`	"The Quarg are an advanced and apparently peaceful alien species that live in a corner of human space, near Tarazed. I honestly don't know that much else about them, but I've heard they're happy to talk to anybody who visits them. You could try that if you want to know more."`
+				to display
+					not "First Contact: Quarg: offered"
+			`	"There's a planet they control that has too much gravity for them, but is just right for humans, so they let humans settle there under their protection. Or at least that's what I've heard."`
+				goto "main choice"
+	on visit
+		dialog phrase "generic cargo and passenger on visit"
+	on fail
+		dialog "Penny Little and Rosa Perkins let you know that they wanted to get to <destination> to avoid the hostilities, so they'll find another captain who's still neutral."
```

Comment:
```suggestion
		dialog "Penny and Rosa let you know that they wanted to get to <destination> to avoid the hostilities, so they'll find another captain who's still neutral."
```

Reply:
There's no need to name their full names every time. The first would've been enough.

Hell, there _may_ not even be a need to give their surnames, but that's a topic I'm not keen to go down on.

Reply:
Makes sense, done.